src/Android/Avalonia.Android/Platform/Input/AvaloniaInputConnection.cs

```diff
@@ -147,6 +147,26 @@ public bool PerformEditorAction([GeneratedEnum] ImeAction actionCode)
                     }
             }
 
+            var eventTime = SystemClock.UptimeMillis();
+            SendKeyEvent(new KeyEvent(eventTime,
```

Comment:
I'm not familiar enough with the Android IME, but shouldn't this only be done for `ImeAction.Done`?

Reply:
The previous behavior was done for all actions. We could limit it to only actions that expect the keyboard to be dismissed.